howto/tpus.md

```diff
@@ -17,6 +17,14 @@ cd lit-parrot
 pip install -r requirements.txt
 ```
 
+Since Lit-Parrot requires a torch version newer than torch 2.0.0, we need to manually install nightly builds of torch, torch_xla, and torchvision:
```

Comment:
```suggestion
Since Lit-Parrot requires a torch version newer than torch 2.0.0, we need to manually install nightly builds of torch and torch_xla:
```